app/views/claims/schools/users/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: claims_school_users_path(@school)) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l"><%= @school.name %></span>
+      <h2 class="govuk-heading-l"><%= @user.full_name %></h2>
+
+      <dl class="govuk-summary-list">
```

Comment:
Again, let's use the govuk_summary list component. See [here](https://github.com/DFE-Digital/itt-mentor-services/blob/7758930db2dbabbf75e05d07541e45c75dd78999/app/views/placements/support/organisations/users/_details.html.erb)

And better yet, move that component to a the shared folder and just use it directly so we don't have so much repetition.